packages/notifications/src/Notification.php

```diff
@@ -86,6 +94,11 @@ public static function fromArray(array $data): static
         return $static;
     }
 
+    protected static function isViewSafe(): bool
+    {
+        return false;
+    }
+
```

Comment:
(Will also need to add a type docblock here for 3.x, @danharrin.)

```suggestion
    protected static function isViewSafe(string $view): bool
    {
        return $view === 'notifications::notification';
    }

```